{
  "Faults": [
    {
      "latitude": 51.55328524587392,
      "type": "LV",
      "reference": "KN6378",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-02-15T16:10:00.000+0000",
      "affectedCustomerCount": 13,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "25b871c0-4129-4221-9f94-46c99daf4b1a",
      "minLongitude": -1.8484,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8450490744354262,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-15T17:28:00.000+0000",
      "estimatedRestoration": "2023-02-16T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6378'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6378",
      "maxLongitude": -1.8432,
      "minLatitude": 51.5525,
      "name": "SN5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN5 6BL",
        "SN5 6ED",
        "SN5 6EE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.8484,
              51.5544
            ],
            [
              -1.8432,
              51.5544
            ],
            [
              -1.8432,
              51.5525
            ],
            [
              -1.8484,
              51.5525
            ],
            [
              -1.8484,
              51.5544
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-15T16:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5544,
      "etrUpdate": true
    },
    {
      "latitude": 51.28371224382871,
      "type": "HV",
      "reference": "KN6550",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "3f7bdc29-3d28-4477-acf2-32a0b570c036",
      "minLongitude": -1.0935,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0926090704770965,
      "resolved": false,
      "py/object": "Fault",
      "Messages": [
        {
          "py/object": "Message",
          "senderName": "tobias.sell@sse.com",
          "senderUserId": "44833855-d575-41a0-92f7-7cd8a3b8dabd",
          "text": "Engineers are responding to a main network fault in the area. We aim to restore power by 01:30.",
          "UUID": "a8cc4a21-2f54-410d-aaf7-a5300c46fa74",
          "py/class": "http://opengrid.com/givr/1.0#//Message",
          "sentDateTime": "2023-02-15T22:48:07.455+0000"
        }
      ],
      "srtUpdate": true,
      "loggedAt": "2023-02-15T22:25:00.000+0000",
      "estimatedRestoration": "2023-02-16T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6550'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6550",
      "maxLongitude": -1.0891,
      "minLatitude": 51.2831,
      "name": "RG24 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG24 9DX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0935,
              51.2847
            ],
            [
              -1.0891,
              51.2847
            ],
            [
              -1.0891,
              51.2831
            ],
            [
              -1.0935,
              51.2831
            ],
            [
              -1.0935,
              51.2847
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2847,
      "etrUpdate": true
    },
    {
      "latitude": 51.58331544836443,
      "type": "LV",
      "reference": "KN6524",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-02-15T22:30:00.000+0000",
      "affectedCustomerCount": 34,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "0df8e3bb-7709-41b0-ac95-4f20747211c4",
      "minLongitude": -2.101,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.097068301584844,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-15T20:49:00.000+0000",
      "estimatedRestoration": "2023-02-16T12:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6524'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6524",
      "maxLongitude": -2.0942,
      "minLatitude": 51.5811,
      "name": "SN16 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN16 0AG",
        "SN16 9AS",
        "SN16 9AT",
        "SN16 9AU",
        "SN16 9AX",
        "SN16 9AZ",
        "SN16 9BD",
        "SN16 9BE",
        "SN16 9BG",
        "SN16 9BH",
        "SN16 9BQ",
        "SN16 9BU",
        "SN16 9BY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.101,
              51.5858
            ],
            [
              -2.0942,
              51.5858
            ],
            [
              -2.0942,
              51.5811
            ],
            [
              -2.101,
              51.5811
            ],
            [
              -2.101,
              51.5858
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-15T22:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5858,
      "etrUpdate": true
    },
    {
      "latitude": 51.28274056281743,
      "type": "HV",
      "reference": "KN6586",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-02-16T01:45:00.000+0000",
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "52dc1de4-4bfe-4334-8b39-75b6e4959734",
      "minLongitude": -2.0036,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.9975992184562663,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-16T00:47:00.000+0000",
      "estimatedRestoration": "2023-02-16T09:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6586'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6586",
      "maxLongitude": -1.9896,
      "minLatitude": 51.2806,
      "name": "SN10 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SN10 4BE",
        "SN10 4BQ",
        "SN10 4EU",
        "SN10 4EY",
        "SN10 4HJ",
        "SN10 4HL",
        "SN10 4HN",
        "SN10 4HP",
        "SN10 4HQ",
        "SN10 4HW",
        "SN10 4HX",
        "SN10 4HZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0036,
              51.2858
            ],
            [
              -1.9896,
              51.2858
            ],
            [
              -1.9896,
              51.2806
            ],
            [
              -2.0036,
              51.2806
            ],
            [
              -2.0036,
              51.2858
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-16T02:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2858,
      "etrUpdate": true
    },
    {
      "latitude": 51.17382206118972,
      "type": "LV",
      "reference": "KN6646",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b94f10f6-ab04-4c53-b1c1-0c9012a20500",
      "minLongitude": -1.5909,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.5770717957673581,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-16T04:16:00.000+0000",
      "estimatedRestoration": "2023-02-16T09:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6646'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6646",
      "maxLongitude": -1.5695,
      "minLatitude": 51.1669,
      "name": "SP11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SP11 8BN",
        "SP11 8RX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.5909,
              51.1799
            ],
            [
              -1.5695,
              51.1799
            ],
            [
              -1.5695,
              51.1669
            ],
            [
              -1.5909,
              51.1669
            ],
            [
              -1.5909,
              51.1799
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-16T06:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.1799,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-16T06:15:00.000+0000",
      "latitude": 50.768723232117445,
      "estimatedRestoration": "2023-02-16T10:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6682'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6682",
      "reference": "KN6682",
      "etaUpdate": true,
      "affectedCustomerCount": 6,
      "name": "BH23 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH23 8JL",
        "BH23 8JY",
        "BH23 8JZ"
      ],
      "location": {
        "coordinates": [
          -1.7266493932518263,
          50.768723232117445
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2023-02-16T08:15:00.000+0000",
      "UUID": "134e6883-cfd2-41f3-a6b8-29722cc84bee",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.7266493932518263,
      "resolved": false
    },
    {
      "latitude": 50.63992501805057,
      "type": "HV",
      "reference": "KN6713",
      "etaUpdate": true,
      "affectedCustomerCount": 346,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "d1753001-0774-4f83-b305-79d53d54ae9e",
      "minLongitude": -1.4866,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4080632083652165,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-16T08:13:00.000+0000",
      "estimatedRestoration": "2023-02-16T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6713'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6713",
      "maxLongitude": -1.1834,
      "minLatitude": 50.6059,
      "name": "PO30 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO30 3LJ",
        "PO30 4AG",
        "PO30 4BB",
        "PO30 4BP",
        "PO30 4BS",
        "PO30 4BU",
        "PO30 4BY",
        "PO30 4BZ",
        "PO30 4DA",
        "PO30 4DB",
        "PO30 4DD",
        "PO30 4DS",
        "PO30 4DT",
        "PO30 4DU",
        "PO30 4DY",
        "PO30 4DZ",
        "PO30 4EA",
        "PO30 4ED",
        "PO30 4EE",
        "PO30 4EF",
        "PO30 4EG",
        "PO30 4EH",
        "PO30 4EJ",
        "PO30 4EL",
        "PO30 4EP",
        "PO30 4ER",
        "PO30 4ES",
        "PO30 4ET",
        "PO30 4EU",
        "PO30 4EX",
        "PO30 4EY",
        "PO30 4HA",
        "PO30 4HB",
        "PO30 4HE",
        "PO30 4HF",
        "PO30 4HG",
        "PO30 4HQ",
        "PO30 4LD",
        "PO30 4PG",
        "PO30 4PN",
        "PO30 4PW",
        "PO38 2JH",
        "PO38 2LF",
        "PO38 2LG",
        "PO38 2LH",
        "PO38 2LN",
        "PO38 2LQ",
        "PO38 2LR",
        "PO38 2LW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.4866,
              50.7004
            ],
            [
              -1.1834,
              50.7004
            ],
            [
              -1.1834,
              50.6059
            ],
            [
              -1.4866,
              50.6059
            ],
            [
              -1.4866,
              50.7004
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-16T08:40:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7004,
      "etrUpdate": true
    },
    {
      "latitude": 57.156022467144815,
      "type": "LV",
      "reference": "KN6702",
      "etaUpdate": true,
      "affectedCustomerCount": 17,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "c52f2c0e-2ffc-480b-a610-8749adc585c8",
      "minLongitude": -2.1538,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1519622128218865,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-16T07:49:00.000+0000",
      "estimatedRestoration": "2023-02-16T11:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN6702'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN6702",
      "maxLongitude": -2.15,
      "minLatitude": 57.155,
      "name": "AB16 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB16 5NG",
        "AB16 6XS",
        "AB16 6XT",
        "AB16 6XU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.1538,
              57.1568
            ],
            [
              -2.15,
              57.1568
            ],
            [
              -2.15,
              57.155
            ],
            [
              -2.1538,
              57.155
            ],
            [
              -2.1538,
              57.1568
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-16T08:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.1568,
      "etrUpdate": true
    },
    {
      "latitude": 57.469276998083785,
      "type": "LV",
      "reference": "KN5641",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "b7849365-022f-4e9c-a455-f04ac260f38a",
      "minLongitude": -4.1988,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.190248684039355,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-02-14T11:28:00.000+0000",
      "estimatedRestoration": "2023-02-16T11:15:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KN5641'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KN5641",
      "maxLongitude": -4.1694,
      "minLatitude": 57.4577,
      "name": "IV2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV2 3SY",
        "IV2 3TW",
        "IV2 3UP",
        "IV2 5BG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.1988,
              57.4736
            ],
            [
              -4.1694,
              57.4736
            ],
            [
              -4.1694,
              57.4577
            ],
            [
              -4.1988,
              57.4577
            ],
            [
              -4.1988,
              57.4736
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-02-14T12:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.4736,
      "etrUpdate": true
    }
  ]
}
